apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -241,6 +241,11 @@
                             "description" : "Number of acceptors in internal pool",
                             "type" : "integer",
                             "default" : 10
+                        },
+                        "debug_endpoints" : {
+                            "description" : "Enable (true) debug api. Disabled (false) by default",
```

Comment:
This seems overlapping with:
* "http > endpoints > debug"
* "http > debug"

Could you please detail the schema description for those two config parameters?

Reply:
I updated description to "deprecated" for both. I'm not sure if we should remove them right now or wait for next release.